faker/providers/address/ru_RU/__init__.py

```diff
@@ -384,9 +384,10 @@ def street_name(self):
                 if street in self.street_titles_irregular_masc.keys():
                     result = self.street_titles_irregular_masc[street]
                 else:
-                    result = stem + 'ый'
                     if stem.endswith('ск') or stem.endswith('цк'):
                         result = stem + 'ий'
+                    else:
+                        result = stem + 'ый'
```

Comment:
This change in the file should not actually change the outcome, but I felt was slightly cleaner.